packages/@uppy/react/src/useUppyStateSelector.js

```diff
@@ -0,0 +1,9 @@
+import { useSyncExternalStore } from 'react'
+
+export default function useUppyStateSelector (uppy, selector) {
```

Comment:
```suggestion
export default function useUppyState(uppy, selector) {
```

I would call this `useUppyState`, because:

1. it's simpler 
2. the second argument is the `selector`, which allows you to use a slice of the state. So you are using Uppy state.

The default `selector` should maybe be an identity function, eg. `x => x`?

Reply:
I agree the name should be simpler. I don't want to provide a default selector because Uppy's state rerenders _very_ often and you never want that. I'd rather make it mandatory. The types also force you to return a value of state, rather than the state itself.

Reply:
unless we want to in the future implement another hook called `useUppyState`, I agree it's a nicer name to use